client/js/array-util.js

```diff
@@ -9,7 +9,18 @@ const getLetterRange = (firstletter = 'A', numLetters) => {
            .map((char) => String.fromCharCode(char)); 
```

Comment:
Set your Sublime Text preferences to use 2-space tabs instead of 4-space tabs and to “transform indentations to spaces". This will make sure that future files created use the preferred spacing.

For current files (aka all files in X-Cell), you can adjust the spacing by selecting "Tab Width: 2" and "Convert indentation to spaces" in the bottom right corner. Then save and check in your files again.


This is key for readability, especially with nested components.